tests/components/homekit_controller/fixtures/haa_fan.json

```diff
@@ -4,69 +4,69 @@
     "services": [
       {
         "iid": 1,
-        "type": "0000003E-0000-1000-8000-0026BB765291",
+        "type": "0000003E-0000-1000-8000-0026BB765292",
```

Comment:
I don't think we can change the uuid in the fixture. Thats actual data from the device